src/components/messenger/list/index.tsx

```diff
@@ -238,7 +238,7 @@ export class Container extends React.Component<Properties, State> {
         )}
 
         <FeatureFlag featureFlag='enableRewards'>
-          <div {...cnMessageList('rewards-container', this.props.includeUserSettings && 'center')}>
+          <div {...cnMessageList('rewards-container', !this.props.includeUserSettings && 'center')}>
```

Comment:
For `center` to make sense, the bang needs to be added. This means that if `!this.props.includeUserSettings` is true, which is equal to `isMessengerFullScreen` being false, then use `center` to centre the rewards button when not in full screen.

Before this change it is the opposite.